### DISCUSSION TITLE
Feature Request: minimize comments in a pull request

### DISCUSSION THREAD

**Discussion by YairoR** [community/community#18408] [Pull Requests]
I think that comments in PR should be minimized so it will be easier to continue the review (currently each comment takes lots of space and it's difficult to "ignore" the existing comments). An example of implementation:
![image](https://user-images.githubusercontent.com/7312492/173221129-eb41bf2e-2390-4475-932a-813df9234d4b.png)

**Comment by AlejoAsd** [comment]
I am honestly surprised this isn't a feature already. Some developers like to leave notes in their PRs to clarify some aspect of the code. These are helpful, but get in the way when trying to read the changes.

I would like to be able to hide/show comments in the `Files changed` tab of the PR interface.

**Comment by DavidIbarr** [comment]
I agree this should be a feature, especially for the 'Conversation' section. While it does encourage problems to be addressed that are not currently resolved (which then do get 'minimized') at times you might be waiting for a reply or for a few comments to be otherwise addressed by another person while you are fixing other areas of the code. The current 'hide' functionality does not address this either, because you are telling others that the comment is 'outdated' or 'off-topic,' for example. This would allow you to better manage what is important to you in your own workspace, without affecting others.

**Comment by DavidIbarr** [comment]
Additionally, while you can minimize individual _files_ in the 'Files changed' tab, you can't minimize individual _conversations_ that may be more or less important to you.

**Comment by ettaboyle** [comment]
We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our [public roadmap](https://github.com/github/roadmap/projects/1). While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress.

Regarding your specific request, it’s important to note that it’s not actively being worked on at the moment. However, this doesn’t mean we won’t consider it in the future; it’s simply not on our development roadmap right now.

We’d like to emphasize that as a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please rest assured that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.
While we do review all incoming feedback unfortunately due to the volume of feedback, we may not be able to respond to each one individually at this time. However, we strongly encourage you to continue providing your valuable input as it greatly helps us prioritize and improve GitHub.

Your contributions are instrumental in making GitHub better for all users. Thank you for taking the time to share your thoughts with us, even if the particular request is not currently in active development. Your input is truly valuable and contributes to the continuous improvement of GitHub.
